### PR TITLE
Guide changes

### PR DESCRIPTION
**Pull request by m1-dev**
- Used `Function` type instead of `void` because that's more descriptive, especially for beginners.
- Added missing `SlashCommandBuilder` imports for the guide's autocomplete examples.

### CODE REVIEW

File: apps/docs/pages/guide/command-file-setup.mdx [hunk 1]
```diff
@@ -290,15 +293,15 @@ This property contains the command's structural information, and is required to
 
 ### `run`
 
--   Type: `void`
+-   Type: `Function`
```
Comment (twlite) [inline apps/docs/pages/guide/command-file-setup.mdx:296]: this seems to be the return type of `run()`

File: apps/docs/pages/guide/command-file-setup.mdx [hunk 2]
```diff
@@ -290,15 +293,15 @@ This property contains the command's structural information, and is required to
...
 
 -   Props Type: [`SlashCommandProps`](/docs/typedef/SlashCommandProps) | [`ContextMenuCommandProps`](/docs/typedef/ContextMenuCommandProps)
 
 This function will be called when the command is executed.
 
 ### `autocomplete`
 
--   Type: `void`
+-   Type: `Function`
```
Comment (twlite) [inline apps/docs/pages/guide/command-file-setup.mdx:304]: same here

**Review by twlite — CHANGES_REQUESTED** [review]

File: apps/docs/pages/guide/command-file-setup.mdx (hunk 1 above)
Comment (m1-dev) [reply apps/docs/pages/guide/command-file-setup.mdx:296]: as I said, beginners (to javascript) can be confused with the type

**Comment by m1-dev** [comment]
I think I messed something up... I'll open another PR when I find out what I did.

**Pull request closed without merging**